test/clj_3df/core_test.cljc

```diff
@@ -92,6 +94,7 @@
 (deftest test-cartesian
   (let [name "cartesian"
         db   (df/create-db {:name {:db/valueType :String}})]
+    (exec! (debug-conn) (create-db-inputs db))
```

Comment:
Thanks for updating those, just noticed a few days ago...
Any reason for separate `exec!`'s? I think it should be fine to do

``` clojure
(exec! (debug-conn)
  (create-db-inputs db)
  (register-plan  ...)
  ...)
```